### PR TITLE
Update OCC build

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: packages/jupytercad-opencascade/lib/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import initOpenCascade from 'opencascade.js';
```
Comment (martinRenou) [inline packages/jupytercad-opencascade/lib/index.js:1]: Try to get rid of these as typings are not forwarded properly and not visible in VSCode

**Comment by martinRenou** [comment]
Remaining TODO:
- Fix CI (pydantic issue?)
- Get rid of manual `lib/index.js` and `lib/index.d.ts`
- Get rid of the full WASM build file from dist

**Comment by martinRenou** [comment]
@trungleduc marking as ready for review! But as discussed, let's merge after your refactor PR

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks @martinRenou!

**Comment by martinRenou** [comment]
Doesn't build anymore since the rebase
I need to push a commit

**Comment by martinRenou** [comment]
Back to the check-release and integration tests failures, let's merge

**Pull request merged**